src/XRP/default-xrp-client.ts

```diff
@@ -445,6 +447,43 @@ export default class DefaultXrpClient implements XrpClientDecorator {
     return await this.getTransactionResult(transactionHash)
   }
 
+  /**
+   * Enables DepositPreauth and authorizes an XRPL account to send to this XRPL account.
+   *
+   * @set https://xrpl.org/depositpreauth.html
```

Comment:
typo - I assume this is supposed to be `@see`

Reply:
got it